lib/manager/npm/post-update/yarn.spec.ts

```diff
@@ -242,85 +246,62 @@ describe('manager/npm/post-update/yarn', () => {
   );
 
   it('catches errors', async () => {
+    Fixtures.mock({});
     const execSnapshots = mockExecAll(exec, {
       stdout: '1.9.4',
       stderr: 'some-error',
     });
-    fs.readFile.mockResolvedValueOnce(null).mockRejectedValueOnce(() => {
-      throw new Error('not-found');
-    });
     const res = await yarnHelper.generateLockFile('some-dir', {});
-    expect(fs.readFile).toHaveBeenCalledTimes(2);
```

Comment:
whay we can't have that check?

Reply:
Returned, previous implementation did not allowed it